code/modules/client/preference/preferences.dm

```diff
@@ -88,7 +88,6 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 		"1017" = 100, // CHANNEL_ENGINE
 		"1016" = 100, // CHANNEL_FIREALARM
 		"1015" = 100, // CHANNEL_ASH_STORM
-		"1014" = 100, // CHANNEL_DELTA_ALARM
```

Comment:
Will this have any problems with the database? If someone set the channel 1014 to 5, and then in the future the game selects 1014 as a random channel, would the sound be randomly very quiet?

Reply:
I can run a script to wipe out the 1014 entry.